js/ui/switch.js

```diff
@@ -76,15 +73,7 @@ const Switch = Editor.inherit({
                 options: {
                     focusStateEnabled: true
                 }
-            },
-            {
-                device: function(device) {
-                    return themes.isIos7(themeName);
```

Comment:
Should we rework this behavior dependent on `platform == ios`?
/cc @DokaRus

Reply:
Nope. This is theme related config
We can remove `_animateHandle` option at all. It only makes sense for the ios7 theme.

Reply:
@DokaRus  I have removed this option. Look, please.

Reply:
Ok